app.js

```diff
@@ -1,54 +1,73 @@
-//Button for the form section
-let addButton = document.querySelector(".submit");
-// Div element for the bookshelf
-let bookshelf = document.querySelector(".bookshelf");
+// Button for the form section
+
+// /* eslint-disable */
 
-let title = document.getElementById("title");
-let author = document.getElementById("author");
+const addButton = document.querySelector('.submit');
+
+// Div element for the bookshelf
+const bookshelf = document.querySelector('.bookshelf');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
 
 // The collection that holds the author and the title
-let books = [];
+const books = [];
 
+// Function template that creates the 'p' elements
 function creation(element, text, parent) {
-  let item = document.createElement(element);
+  const item = document.createElement(element);
   item.innerHTML = text;
   parent.appendChild(item);
 }
 
-addButton.addEventListener("click", function () {
-  creation("p", title.value, bookshelf);
-  creation("p", author.value, bookshelf);
-
-  let removeButton = document.createElement("button");
-  removeButton.innerHTML = "remove";
-  bookshelf.appendChild(removeButton);
-
-  
+// Event listener that appends the button to the html element
+/* eslint-disable */
+addButton.addEventListener('click', () => {
+  if (title.value !== '' || author.value !== '') {
+    const wrapper = document.createElement('div');
+    bookshelf.appendChild(wrapper);
+    creation('p', title.value, wrapper);
+    creation('p', author.value, wrapper);
+    createRemove('remove', wrapper);
+    localStorages();
+    title.value = '';
+    author.value = '';
+  }
+});
 
+// Function that stores the data to the local storage
+function localStorages() {
   books.push({ Title: title.value, Author: author.value });
+  const stringified = JSON.stringify(books);
+  localStorage.setItem('data', stringified);
+}
 
-  let stringified = JSON.stringify(books);
-
-  localStorage.setItem("data", stringified);
-
-  title.value = "";
-  author.value = "";
-});
+// Function that removes the data when the user presses the remove button
+function createRemove(text, parent) {
+  const removeButton = document.createElement('button');
+  removeButton.textContent = text;
+  removeButton.addEventListener('click', () => {
+    bookshelf.removeChild(parent);
+    localStorage.removeItem('data');
+  });
+  parent.appendChild(removeButton);
+}
```

Comment:
- [ ] As suggested by the previous reviewer, kindly work on your remove functionality, when I remove a book, all books gets removed from the local storage and when I reload the page all books get removed from the user interface as well. Kindly check my [loom video](https://www.loom.com/share/a1e6bede646846cc935d5f0649a506c7) here for more explanation:


https://www.loom.com/share/a1e6bede646846cc935d5f0649a506c7

Reply:
Thank you for the clarification. I will fix that right away.